sdks/java/io/mongodb/src/main/java/org/apache/beam/sdk/io/mongodb/MongoDbGridFSIO.java

```diff
@@ -57,8 +61,10 @@
  * <p>
  * <p>MongoDbGridFSIO source returns a bounded collection of String as {@code PCollection<String>}.
```

Comment:
This is not true, it returns `PCollection<T>`. Please audit the rest of this comment for copy-paste leftovers as well.